### PR TITLE
Contact form1

### PR DESCRIPTION
**Pull request by stenlisuryadinata**
Contact form1

### CODE REVIEW

**Comment by danielamoreno699** [comment]
**Mobile portafolio project Contact form**

_In this milestone project, We:_

-Added more HTML elements 
-Added CSS style
-Added validations to the inputs ( 30 characters to the name and 500 characters to textarea and added min length to input fields)
-Added placeholders
-Added the url of the server to send the data
-Improved the style and added hovers

File: src/index.html [hunk 1]
```diff
@@ -241,6 +243,41 @@ <h2>About <br>Myself<br></h2>
...
 </section>
 
+<section class="contact-form">
+
+    <div class="form-container">
+        <div class="form-text">
+            <h2 >contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action= "https://formspree.io/f/mgebklj" method="post">
```
Comment (didierganthier) [inline src/index.html:253]: Well done on adding `formspree` 👍🏽 but something is probably missing in the `ID` or this is just not the right one 🤷🏽 , I get this when I submit the form 👇🏽 
Screenshot 2023-02-21 at 20 00 47

File: src/index.html [hunk 2]
```diff
@@ -241,6 +243,41 @@ <h2>About <br>Myself<br></h2>
...
+            <ul>
+                <li>
+                  <label for="username"></label>
+                  <input type="text" id="username" name="username" placeholder="type name" required maxlength="30" minlength = "5" />
+                </li>
+                <li>
+                  <label for="email"></label>
+                  <input type="email" id="email" name="email" placeholder="type email" minlength = "5" />
```
Comment (didierganthier) [inline src/index.html:261]: Validation makes your form better 😄 so to improve this all input fields should have the `required` attribute to activate the validation on submit. You already have it on the other ones but you must also add it to this one. 😉

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @danielamoreno699 and @stenlisuryadinata ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [ ] Nice team 👥 so let's make sure everyone gets credit by updating the README and adding both contributor's info
- [ ] Talking about README 📄 make sure that all of the required sections are updated, the FAQ is optional so if you're not going to update it you should remove it. 😉 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: README.md [hunk 3]
```diff
@@ -205,22 +205,26 @@ Example:
...
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-
-
 - [ ] **[new_feature_1]**
 - [ ] **[new_feature_2]**
 - [ ] **[new_feature_3]**
```
Comment (IngDavidTM) [inline README.md:230]: - [ ] While you did a wonderful job implementing the changes suggested by the previous reviewer :+1: there is something that you missed. Kindly as requested before kindly make sure that all of the required sections from the README file are updated. It means you should add `Key Features`, `Fututre Features`, and all the `Getting started` sections :+1:

**Review by IngDavidTM — COMMENTED** [review]
## Hi @team 🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by danielamoreno699** [comment]
Hello, I accidentally merged the branch because my coding partner made changes on the main branch so I though that merging was the only solution

**Comment by didierganthier** [comment]
Hi @danielamoreno699, I'd suggest you create another Pull Request and add a link to this one in the description. You may also provide the explanation you provided here, then request another review on the dashboard. Best of luck 🤞

**Review by mckent05 — COMMENTED** [review]
Hi @stenlisuryadinata, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**